### PR TITLE
Beautify the code

### PR DESCRIPTION
**Pull request by lurr**
Changes in this pull:
- Change `var` to `let` in as many cases as allowed
- Beautify the code.
- Literalize specific strings, instead of using primitive concatenation.
- Add `use strict` at the top of files without.
- Change `==` to `===` and `!=` to `!==`

### CODE REVIEW

**Comment by tedivm** [comment]
. . . so did you test it?

**Comment by lurr** [comment]
Test successful!

File: src/roles/upgrader.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
+'use strict'
+
 const MetaRole = require('roles_meta')
 
-const CONTROLLER_MESSAGE = "* Self Managed Code * quorum.tedivm.com * #quorum in Slack *"
+const CONTROLLER_MESSAGE = '* Self Managed Code * quorum.tedivm.com * #quorum in Slack * (Lur was here)'
```
Comment (bastianh) [inline src/roles/upgrader.js]: nice try

**Comment by bastianh** [comment]
why do you hate semicolons? and why do you replace var's with let but not use const when appropriate?
I would not replace var with let inside loops.. if you don't want any var in your code you could define the variables outside of the loop.

File: src/roles/upgrader.js (hunk 1 above)
Comment (tedivm) [reply src/roles/upgrader.js]: Yup, this is why i'm downvoting. Come on @lurr, cut the trolling 😛

**Comment by tedivm** [comment]
To be clear I'll change my upvote once the controller message is fixed.

File: src/roles/upgrader.js (hunk 1 above)
Comment (ags131) [reply src/roles/upgrader.js]: I'll also upvote once this is corrected 😉

**Comment by lurr** [comment]
Changed.

**Pull request merged**